### PR TITLE
Add appointments to person

### LINKED ISSUE #116: Add appointments to person
We should have a feature to add the ability to set appointments for person

### PR DESCRIPTION
**Pull request by SuhailLoya**
fixes #116 

adds an appointment date field to person and a setappointment command to set / unset appointment date

### CODE REVIEW

**Comment by SuhailLoya** [comment]
tests will be added by #118

File: src/main/java/careconnect/model/person/AppointmentDate.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+import careconnect.commons.util.AppUtil;
+import careconnect.logic.parser.exceptions.ParseException;
+
+/**
+ * Represents a Person's Appointment Date in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidAppointmentDateString(String)}
+ */
+public class AppointmentDate {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Appointment Dates must be valid and in the format yyyy-MM-dd";
```
Comment (AppleJem) [inline src/main/java/careconnect/model/person/AppointmentDate.java:18]: For next iteration we can add time! Esp because the meeting logs have timestamp

**Review by AppleJem — APPROVED** [review]
Naisuuuuuuu just tiny nitpick

File: src/main/java/careconnect/logic/parser/ParserUtil.java [hunk 2]
```diff
@@ -150,6 +151,24 @@ public static Date parseLogDate(String date) throws ParseException {
...
     }
 
+    /**
+     * Parses a {@code String dateString} into a {@code AppointmentDate}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code dateString} is invalid.
+     */
+    public static AppointmentDate parseAppointmentDate(String dateString) throws ParseException {
+        if (dateString.isEmpty() || dateString == null) {
```
[the remark this answers is not in the document]
Comment (SuhailLoya) [reply src/main/java/careconnect/logic/parser/ParserUtil.java]: Thanks for catching! fixed!

**Pull request merged**